### DISCUSSION TITLE
Changes in SUMMARY.MD not reflected in app.gitbook.com

### DISCUSSION THREAD

**Discussion by rkranendonk** [GitbookIO/community#714] [1. How to]
I have changed the contents of SUMMARY.MD in my Github repository, but this did not cause visible changes to the content on Gitbook. What could be the cause and how can I solve this?

**Comment by aikoii** [comment]
👋 @rkranendonk! We'll need to check our logs. Would you mind sending an email to support@gitbook.com with your space ID or name
